src/main/scala-sbt-0.13/sbtprotoc/Compat.scala

```diff
@@ -0,0 +1,16 @@
+package sbtprotoc
+
+private[sbtprotoc] trait Compat { self: ProtocPlugin.type =>
+  protected def makeArtifact(f: protocbridge.Artifact): ModuleID = {
+    import Keys.{crossVersion => _}
+    ModuleID(f.groupId, f.artifactId, f.version, crossVersion =
+      if (f.crossVersion) CrossVersion.binary else CrossVersion.Disabled)
+  }
+
+  protected object CacheArguments {
```

Comment:
The name has to be changed because Scalac does not allow definition of companions in other source files than the classes.